From: Semen Kreyda <aswerklon@gmail.com>
Date: Thu, Apr 18 2019 21:01:56
Subject: [PATCH] ARM: dts: sunxi: H2+: enable soc-audio

sunxi: Allwinner H2+ (Orange Pi Zero)
enable built-in soc-audio in bootloader supporting
---
 arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts | 5 +++++

--- a/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
+++ b/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
@@ -150,6 +150,11 @@
 &ohci1 {
 	status = "okay";
 };
+
+&codec {
+	allwinner,audio-routing = "Line Out", "LINEOUT", "MIC1", "Mic", "Mic", "MBIAS";
+	status = "okay";
+};
 
 &spi0 {
 	/* Disable SPI NOR by default: it optional on Orange Pi Zero boards */
